cmd/podman/volumes/prune.go

```diff
@@ -58,7 +59,18 @@ func prune(cmd *cobra.Command, args []string) error {
 	}
 	if !force {
 		reader := bufio.NewReader(os.Stdin)
-		fmt.Println("WARNING! This will remove all volumes not used by at least one container.")
+		fmt.Println("WARNING! This will remove all volumes not used by at least one container. The following volumes will be removed:")
+		listOptions.Filter, err = filters.ParseFilterArgumentsIntoFilters(filter)
```

Comment:
This would list all volumes, not just the ones that would be removed.  podman volume prune is going to prune the volumes that are not in use. 

I believe you would need to have a filter to show volumes that are not in use.

Reply:
Hi @rhatdan.
Yes, you are right. Didn't consider the actual use of the `prune` command. 
Now I added a function in `cmd/podman/volumes/prune.go` in order to get the used volumes and remove them from the available ones for pruning.